README.md

```diff
@@ -11,90 +11,36 @@
 A command line utility to talk to a tchannel server.
 
 ```
-tcurl [-H <hostlist> | -p host:port] <service> <endpoint> [options]
-
-  Version: 4.14.0
-
-  Options:
-    --head (-2) [data] JSON or raw
-    --body (-3) [data] JSON or raw
-      (JSON promoted to Thrift via IDL when applicable)
-    --shardKey send ringpop shardKey transport header
-    --depth=n configure inspect printing depth
-    --thrift (-t) [dir] directory containing Thrift files
-    --no-strict parse Thrift loosely
-    --json (-j) Use JSON argument scheme
-      (default unless endpoint has ::)
-    --http method
-    --raw encode arg2 & arg3 raw
-    --health
-    --timeout [num]
+usage: tcurl [OPTIONS] service endpoint
+
+Options:
+  -h --help                 Show detailed manpage
+  -v --version              Print version
+  -H --hostlist             Path to hostlist file
+  -p --peer                 IP and port of single peer
+  -t --thrift               Path to thrift IDL file
```

Comment:
or directory containing IDL files. Aside: let’s keep in mind we might someday support passing the path to an `idl` directory too.